ranger/container/bookmarks.py

```diff
@@ -185,7 +185,13 @@ def save(self):
             old_perms = os.stat(self.path)
             os.chown(path_new, old_perms.st_uid, old_perms.st_gid)
             os.chmod(path_new, old_perms.st_mode)
-            os.rename(path_new, self.path)
+
+            if os.path.islink(self.path):
+                link_path = os.path.realpath(self.path)
```

Comment:
Wouldn't `target_path` be a clearer name for this? It's the link's *target* we want to write to, right?

Reply:
It would indeed.
`link_path` does kind of hints towards the path where is link is located.

Reply:
I've made the change, travis should be done shortly.